### PR TITLE
Optional Autoescaping in HeadMeta ViewHelper

### PR DESCRIPTION
**Pull request by moegeler**
Bugfixing for https://github.com/zendframework/zend-view/issues/180

### CODE REVIEW

**Comment by froschdesign** [comment]
@malukenho 
Do you really think that the change from `assertEquals` to `assertNotEquals` is correct? Even if the result is the same and true in both tests? 😉

File: test/Helper/HeadMetaTest.php [hunk 1]
```diff
@@ -619,6 +619,6 @@ public function testConditionalNoIEWidthSpace()
     public function testTurnOffAutoEscapeDoesNotEncode()
     {
         $this->helper->setAutoEscape(false)->appendHttpEquiv('foo', 'bar=baz');
-        $this->assertEquals('<meta http-equiv="foo" content="bar=baz" />', $this->helper->toString());
+        $this->assertNotEquals('<meta http-equiv="foo" content="bar=baz">', $this->helper->toString());
```
Comment (froschdesign) [inline test/Helper/HeadMetaTest.php]: This change makes no sense. Please check for the _same_ result.

File: src/Helper/HeadMeta.php [hunk 2]
```diff
@@ -243,7 +243,7 @@ public function itemToString(stdClass $item)
             if (! in_array($key, $this->modifierKeys)) {
                 continue;
             }
-            $modifiersString .= $key . '="' . $this->escapeAttribute($value) . '" ';
+            $modifiersString .= $key . '="' . (($this->autoEscape) ? $this->escapeAttribute($value) : $value) . '" ';
```
Comment (froschdesign) [inline src/Helper/HeadMeta.php]: Please use `sprintf` for better reading here.

File: src/Helper/HeadMeta.php [hunk 3]
```diff
@@ -271,8 +271,8 @@ public function itemToString(stdClass $item)
         $meta = sprintf(
             $tpl,
             $type,
-            $this->escapeAttribute($item->$type),
-            $this->escapeAttribute($item->content),
+            ($this->autoEscape) ? $this->escapeAttribute($item->$type) : $item->$type,
```
Comment (froschdesign) [inline src/Helper/HeadMeta.php]: The first brackets can be omitted.

**Comment by moegeler** [comment]
Sorry, that was my first pull-request. I hope that it is alright now.

**Comment by weierophinney** [comment]
Thanks, @moegeler!

**Pull request merged**